templates/service-template.yml

```diff
@@ -395,19 +406,37 @@ objects:
     metadata:
       name: maestro-grpc
       labels:
-        app: maestro-grpc
+        app: maestro
         port: grpc
       annotations:
         description: Exposes and load balances the maestro pods grpc endpoint
-        service.alpha.openshift.io/serving-cert-secret-name: maestro-tls
     spec:
       selector:
         app: maestro
       ports:
-        - port: 8090
+        - name: grpc
+          port: 8090
```

Comment:
Can we use ${GRPC_SERVER_BINDPORT}?

Reply:
After changing to `${GRPC_SERVER_BINDPORT}`, I encountered the following issue:
```
json: cannot unmarshal string into Go struct field ServicePort.spec.ports.port of type int32
```